NewForce-Cohort-7/tabloidmvc-mollasses-gang
# Added the ability to view a list of all users.

Description
Added the feature for an admin to view a list of all user profiles. Given that a user is logged in and selects the Users they should be directed to a list view of al the current users. 

Testing Instructions

1.  git add and git commit whatever you're working on on your branch
2.  git checkout main
3.  git fetch --a
4.  git checkout mc1
5.  If you don't see the most recent version of my files, git pull origin <mc1>
6.  If you haven't created the Tabloid MVC database, please do so.
7.  Use this to fill out tables and columns: [SQL data] (https://github.com/NewForce-Cohort-7/tabloidmvc-mollasses- 
gang/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data] (https://github.com/NewForce-Cohort-7/tabloidmvc-mollasses-gang/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)

SELECT * FROM UserProfile will display all current users in the database and their details

10. Click the execute solid green button to create database
11. Click the execute with debugger solid green button to launch the app
12.  In browser, please go to Login on the nav bar and log in with this email: admin@example.com
13. This should redirect you back to the home page.
14. User should be showing on the nav bar now. Please click this to view a list of users.
15. You will see 3 | FullName | DisplayName |  UserTypeId 
16. To the right of full name you will see links for (Edit, Details, Delete)
17. Log out should be showing on the nav bar now. Please click this to log out.





Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
